src/V3Ast.cpp

```diff
@@ -62,8 +62,8 @@ std::ostream& operator<<(std::ostream& os, VNType rhs);
 //######################################################################
 // VSelfPointerText
 
-const std::shared_ptr<const string> VSelfPointerText::s_emptyp = std::make_shared<string>("");
-const std::shared_ptr<const string> VSelfPointerText::s_thisp = std::make_shared<string>("this");
+const string VSelfPointerText::s_emptyp = "";
```

Comment:
Remove the "p" suffixes when no longer a pointer.

Reply:
Thanks, solved now!